dtfs-central-api/src/cron-scheduler-jobs/delete-correction-request-transient-form-data/index.ts

```diff
@@ -0,0 +1,27 @@
+import { asString, CronSchedulerJob } from '@ukef/dtfs2-common';
+import { FeeRecordCorrectionRequestTransientFormDataRepo } from '../../repositories/fee-record-correction-request-transient-form-data-repo';
+
+const { RECORD_CORRECTION_TRANSIENT_FORM_DATA_DELETE_SCHEDULE } = process.env;
+
+/**
+ * Gets and deletes record correction request transient form data if older than 1 day
+ * gets all from the database which are older than 1 day
+ * returns if none found
+ * gets an array of feeIds
+ * deletes transient form data based on provided ids
+ */
```

Comment:
Just wondering if this needs to be so detailed, could we not just have
```suggestion
/**
 * Deletes record correction request transient form data more than 1 day old
 */
```

The details about 'getting' are the concern of the `deleteByLastUpdatedOlderThanOneDayAgo` repo method, which in turn are the concern of typeorm's `delete` method

Reply:
done